projects/pt1/python/torch_mlir_e2e_test/test_suite/norm_like.py

```diff
@@ -243,6 +243,48 @@ def NativeBatchNormNoneWeightModule_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class GroupNormModule(torch.nn.Module):
```

Comment:
No need to have this test. The goal of these e2e tests is to make sure each individual op works as intended. This is achieved by calling the op through `torch.ops.aten...`. When you use `torch.nn....`, PyTorch will often generate extra logic (code for checking inputs, dispatching, etc) that can lead to failures in torch-mlir unrelated to the op being added (I believe this is what is happening here). Using the `nn` module version also makes it harder from looking at the test to know what Torch-MLIR op it corresponds to.

More info on this here: https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#forward-method